### PR TITLE
build(misc): bump kraken-app-portal version 

### PR DESCRIPTION
**Pull request by KsiBart**
## Description
Upgrade versions from downstream v0.188.0 to v0.230.0.

1. **Feature Update:** Separated location and property name fields on Mapping page; users can now update property location details.
2. **Admin Functionality:** Enabled deletion of unused seller API servers for streamlined management.
3. **Buyer Management:** Added functionality to deactivate active buyers, rotated buyer tokens for security, and improved buyer management UI.
4. **Mapping Template:** Released the initial version of the upgraded mapping template.
5. **UI Enhancements:** Addressed responsive design issues for better user experience across devices.

## Types of changes
Add features and fix bugs

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] CI/CD or documentation update (changes to CI/CD pipeline or documentation)

## Checklist

- [x] I ensured that the documentation is up to date
- [x] I explained why this PR updates in detail with reasoning why it's required
- [x] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

File: kraken-app/kraken-app-portal/package.json [hunk 1]
```diff
@@ -1,30 +1,38 @@
 {
-  "name": "kraken-app-portal",
+  "name": "kraken-portal",
   "private": true,
-  "version": "1.0.2",
+  "version": "0.230.0",
```
Comment (KsiBart) [inline kraken-app/kraken-app-portal/package.json:4]: @DaveXiong TBD which version number to include.

File: kraken-app/kraken-app-portal/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+RUN apt-get update \
+    && apt-get upgrade -y \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && chown -R 11020:11020 /usr/share/nginx/html/*
+
+COPY nginx.conf /etc/nginx/conf.d/defalt.conf
+COPY bin/entrypoint.sh /docker-entrypoint.d/react-entrypoint.sh
+
+USER 11020
```
Comment (KsiBart) [inline kraken-app/kraken-app-portal/Dockerfile:38]: Do we need Dockerfile?

File: kraken-app/kraken-app-portal/package.json (hunk 1 above)
Comment (DaveXiong) [reply kraken-app/kraken-app-portal/package.json:4]: `1.0.2` will be used at the moment, pls take note `kraken` will use the same version for FE and BE.
And `version` will be increased manually by the admin.

File: kraken-app/kraken-app-portal/Dockerfile (hunk 2 above)
Comment (DaveXiong) [reply kraken-app/kraken-app-portal/Dockerfile:38]: Dockerfile is required, and it has been moved to https://github.com/mycloudnexus/kraken/blob/main/docker/app-portal/Dockerfile

File: kraken-app/kraken-app-portal/package.json (hunk 1 above)
Comment (KsiBart) [reply kraken-app/kraken-app-portal/package.json]: @DaveXiong done

**Review by DaveXiong — COMMENTED** [review]
comments added

**Comment by DaveXiong** [comment]
@KsiBart  Can you help to add description on hightlight the features will be supported in the PR?

**Review by DaveXiong — APPROVED** [review]
LGTM

**Pull request merged**